packages/terra-clinical-result/translations/sv.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Did we get translations logged for these?

Reply:
Logged here with final strings needed - https://jira.cerner.com/browse/TRANS-7956